apps/els_lsp/test/els_completion_SUITE.erl

```diff
@@ -1675,44 +1675,49 @@ resolve_application_remote_otp(Config) ->
         <<"write/2">>
     ),
     #{result := Result} = els_client:completionitem_resolve(Selected),
-    Value =
-        case has_eep48(file) of
-            true ->
-                <<
-                    "```erlang\nwrite(IoDevice, Bytes) -> ok | {error, "
-                    "Reason}\nwhen\n  IoDevice :: io_device() | atom(),\n  Bytes ::"
-                    " iodata(),\n  Reason :: posix() | badarg | terminated.\n```\n\n"
-                    "---\n\nWrites `Bytes` to the file referenced by `IoDevice`\\. "
-                    "This function is the only way to write to a file opened in `raw`"
-                    " mode \\(although it works for normally opened files too\\)\\. "
-                    "Returns `ok` if successful, and `{error, Reason}` otherwise\\."
-                    "\n\nIf the file is opened with `encoding` set to something else "
-                    "than `latin1`, each byte written can result in many bytes being "
-                    "written to the file, as the byte range 0\\.\\.255 can represent "
-                    "anything between one and four bytes depending on value and UTF "
-                    "encoding type\\.\n\nTypical error reasons:\n\n* **`ebadf`**  \n"
-                    "  The file is not opened for writing\\.\n\n* **`enospc`**  \n"
-                    "  No space is left on the device\\.\n"
-                >>;
-            false ->
-                <<
-                    "## file:write/2\n\n---\n\n```erlang\n\n  write(File, "
-                    "Bytes) when is_pid(File) orelse is_atom(File)\n\n  write(#file_"
-                    "descriptor{module = Module} = Handle, Bytes) \n\n  write(_, _) "
-                    "\n\n```\n\n```erlang\n-spec write(IoDevice, Bytes) -> ok | "
-                    "{error, Reason} when\n      IoDevice :: io_device() | atom(),"
-                    "\n      Bytes :: iodata(),\n      Reason :: posix() | "
-                    "badarg | terminated.\n```"
-                >>
-        end,
-    Expected = Selected#{
-        documentation =>
-            #{
-                kind => <<"markdown">>,
-                value => Value
-            }
-    },
-    ?assertEqual(Expected, Result).
+    ?assertMatch(
+        #{
+            data := #{module := <<"file">>, function := <<"write">>, arity := 2},
+            documentation := #{kind := <<"markdown">>, value := _}
+        },
+        Result
+    ),
+    #{documentation := #{value := ActualValue}} = Result,
+    case {has_eep48_edoc(), has_eep48(file)} of
+        {true, false} ->
+            %% If this should fail the test we should document requirements for
+            %% the development environment so that OTP has the EEP-48 docs.
+            {skip, "This OTP build does not have EEP-48 docs for the 'file' module"};
```

Comment:
Maybe you want this to fail instead of skipping. I think the docs should be updated to ensure developers have the right stuff locally to run the test suite.

I have OTP 24 and 25 installed via asdf and `code:get_doc(file)` yields a map with docs, but no functions have docs:

```
1> {ok, {docs_v1, 0, erlang, <<"application/erlang+html">>, none, _, ModDocs}} = code:get_doc(file).
2> [{F, A, Doc} || {{function, F, A}, _, _, Doc, _} <- ModDocs].
[{native_name_encoding,0,none},
 {format_error,1,none},
 {pid2name,1,none},
 {get_cwd,0,none},
 {get_cwd,1,none},
 {set_cwd,1,none},
 {delete,1,none},
 {delete,2,none},
 {rename,2,none},
 {make_dir,1,none},
 {del_dir,1,none},
 {del_dir_r,1,none},
 {read_file_info,1,none},
 {read_file_info,2,none},
 {altname,1,none},
 {read_link_info,1,none},
 {read_link_info,2,none},
 {read_link,1,none},
 {read_link_all,1,none},
 {write_file_info,2,none},
 {write_file_info,3,none},
 {list_dir,1,none},
 {list_dir_all,1,none},
 {read_file,1,none},
 {make_link,2,none},
 {make_symlink,2,none},
 {write_file,2,...},
 {write_file,...},
 {...}|...]
3> [{F, A, Doc} || {{function, F, A}, _, _, Doc, _} <- ModDocs, Doc =/= none].
[]
```